src/main/java/duke/command/AddDeadlineCommand.java

```diff
@@ -0,0 +1,52 @@
+package duke.command;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeParseException;
+
+import duke.datetime.DateTime;
+import duke.dukeexception.DukeException;
+import duke.storage.Storage;
+import duke.task.Deadline;
+import duke.tasklist.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Class in charge of handling the case of adding Deadline Task
+ */
+public class AddDeadlineCommand extends Command {
```

Comment:
I like how you have a Command class that branches out to the different commands for duke respectively which follows good OOP principles.